Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.5-alpine
```

Comment:
Any reason why we can't use `ruby:alpine` to get the latest version of ruby? As far as I understand, CeWL works on the latest version.